package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Validate your BPMN diagrams based on configurable lint rules",
   "version": "7.2.0",
   "main": "lib/index.js",
+  "module": "lib/main.js",
```

Comment:
What exactly are these changes about here?

Reply:
I did not change them actively... maybe my IDE did?

Reply:
This is due to the merge you did.

Make sure your feature branch is up-to-date with the upstream (bpmnlint master):

```
git rebase upstream/master
```